rfcs/033-lineage-graph.md

```diff
@@ -0,0 +1,138 @@
+- Feature Name: lineage-graph-visualization
+- Start Date: 2021-04-01
+- RFC PR: [amundsen-io/rfcs#33](https://github.com/amundsen-io/rfcs/pull/33)
+- Amundsen Issue: [amundsen-io/amundsen#0000](https://github.com/amundsen-io/amundsen/issues/0000) (leave this empty for now)
+
+# Amundsen Lineage Stage 1 - Graph Visualization
+
+## Summary
+
+This is the next in the series of https://github.com/amundsen-io/rfcs/pull/25
+
+Implement a navigable canvas-based graph visualization for lineage. Lineage is traditionally represented this way, suboptimally, in the various existing tools like CompilerWorks, Castor, Apache Atlas, etc.
+
+This will eventually replace the existing `Upstream` and `Downstream` tabs on the table detail page (once completely implemented and verified), and introduce a new CTA "Lineage" on the table detail page.
+
+## Motivation
+
+> Why are we doing this? What use cases does it support? What is the expected outcome?
+
+A map graph UI can more fully, transparently, and frictionlessly convey the lineage information. By co-locating data from multiple sources, Amundsen is positioned to create a better map graph UI experience vs. competitors.
+v0 of lineage is already available in the Amundsen project today in the form of a list.
+
+## Guide-level Explanation (aka Product Details)
+
+This feature will expose upstream and downstream tables and columns within the Table Details page, using a navigable graph visualization.
+
+- Lineage: Lineage is a term that describes the flow of data from one entity to another. While this term can broadly include everything from services, events, ETLs, and dashboards, we will focus on table-to-table and column-to-column data lineage in this RFC.
+- Upstream: Upstream is a relative term that describes data sources from which we inherit. Data flows from upstream to downstream.
```

Comment:
maybe this is also opportunity to think about word choice here? upstream/downstream are terms with which you need to spend some time to feel familiar with, they seem very `technical`. for example - in Atlas, the word instead of `downstream` there is `impact` - more closely aligned with what lineage is all about.

Reply:
I'd keep this upstream/downstream for now.